compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -157,8 +157,7 @@ object RefChecks {
    *    1.8.3  M is of type ()S, O is of type []T and S <: T, or
    *    1.9.1  If M is erased, O is erased. If O is erased, M is erased or inline.
    *    1.9.2  If M or O are extension methods, they must both be extension methods.
-   *    1.10   If M is an inline or Scala-2 macro method, O cannot be deferred unless
-   *           there's also a concrete method that M overrides.
+   *    1.10   If O is inline, M must be inline
```

Comment:
`inline` methods should be effectively final

Reply:
We now have a separate issue to discuss & track that.